GoogleChromeLabs/ps-analysis-tool
# Feature: List cookies set via `document.cookie`

## Description
This PR aims to add cookies that were added via the Javascript method using `document.cookie` to be displayed in the extension.

## Relevant Technical Choices
- Added a refresh button like the application tab which refreshes the cookies set by the Javascript method.
- Moved analyse the tab to cover only cookies section other parts such as CHIPS etc are being displayed.
- Handled the extension context invalidated case by showing a message of context being invalidated.

## Testing Instructions
- Open the terminal and run `npm I && npm run dev`.
- Go to travelopia site and open devtools. Go to the application tab and clear all cookies.
- Now reload the page. Go to the Privacy sandbox tab.
- There should not be any cookie. Go to the frame under cookies.
- Click on the refresh button on the side of the search bar.
- It should load all the cookies that were set using document.cookie method

## Screenshot/Screencast



---
